README.md

```diff
@@ -0,0 +1,95 @@
+# Lambda skeleton
+
+## Introduction
+
+This project is designed to make it easier to develop a new AWS Lambda function.
+
+It provides:
+
+- A minimal and empty project which you can base your new Lambda functions on.
+- Dev tools like PHPUnit, PHPStan, CS fixer etc.
```

Comment:
FIX(?): I don't think this is true.

The toolchain is a dev dep so it's only included when you work on this project specifically.  Another project that depends on this will get it's deps but not it's dev deps and thus needs to include the toolchain in it's own dev deps.

Reply:
Hm, I'm not sure. We consider this repo as a template. If you run `composer create-project lambda-skeleton` it'll clone this repository so you will have the same composer.json.

Reply:
Oh I see.

Why that over having it as a dep like any other library?

Reply:
Because if we make it as a library, I guess it'll contain only composer.json so it's useless. Almost all other files in the repo supposed to be editable depending on a function.

Reply:
Yeah, sorry.  I was still picturing this as our own version of Bref.

This makes sense now.  Thanks.